features/FxA-53-send-sms/README.md

```diff
@@ -0,0 +1,444 @@
+
+## Problem Summary and Context
+
+Sync setup through first run via FxA serves the purpose of helping users setup multiple devices and to ensure their data is syncing across their devices.
+
+Unfortunately, within the first 48h of registering for Sync, only 6.4% of users setup a second device and approximately only 8% after 7 days (as of October 30th).
+
+Additionally, approximately only 15% of FxA MAU use their account on two different devices. This value is dropping every month as the number of single device users keep growing. (as of September 30th)
+
+We need to better guide the user through the setup of two devices while removing as much friction as possible. Next steps should always be clear to users.
+
+In order to remove friction, we will provide users with a link to download Firefox via SMS.
+
+Here are the 4 phases we are rolling out in order to resolve this issue:
+
+1. UA detection and smart users flows with no dead ends. We will use download buttons to promote mobile.
+
+2. Integrate SMS download field to email confirmation screen in order when user confirms email on desktop.
+
+3. Deep link the user into the app and pre-populate the email field.
+
+4. Pass a session token or code in the deeplink and avoid any login whatsoever (with a confirmation prompt on originating device)
+
+## Outcomes
+
+The outcome we are looking for is a slick user flow that makes it obvious to users that in order to complete their setup of Sync, they needs to add another device after email confirmation.
+
+## Hypothesis
+
+If we offer to download via SMS to remove friction around setting up a second device for firstrun users on desktop, then we will see an increase in multi-device users within the first 7 days of registration because we have observed that less than [30% of registration](https://next.adjust.com/#/statistics/overview/default/gqrwnbypw28p/hqnqs2?from=2016-10-01&to=2016-10-31&range=last-month&sort=installs&reverse=true) (65k clicks / 210k registration / month) engage with our mobile download buttons and [at best 26%](https://sql.telemetry.mozilla.org/queries/1254#2173) (8% 7d md / 30% ctr) of them follow through to an install within 7 days after clicking.
+
+## Metrics
+
+We will know our hypothesis to be true when we see the proportion of multi-device users increase after 48h and 7d of registration to FxA+Sync on firstrun.
+
+<img src="./image_0.png" / alt="New signups that connect another device" />
+
+From: https://sql.telemetry.mozilla.org/queries/1254#2173
+
+We will also want to keep an eye out for:
+
+* **Email confirmation rate**: we should not impact this negatively in any way.
+
+    * Measured with telemetry and visualized in re:dash
+
+* **Volume of SMS sent**: Will confirm if users are adopting new mechanism.
+
+    * Measured with SMS provider
+
+* **SMS send rate**: Will confirm if users are adopting new mechanism.
```

Comment:
I think it's worth defining more precisely what this is - is it the number of login flows where an SMS is sent, as a percentage of the number of flows that see the SMS form?  Does it map directly to one of the formula listed below?